docs/how-to/default-profile.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Default profile
+legacyUrl: https://docs.ipfs.io/guides/examples/default-profile/
+description: Hands-on guide for rolling back to the default profile
+---
+
+# Configure a default profile and switching between profiles
+
+---
+### What is a profile
+A profile can be used to tweak the performance of your IPFS node. If you want to have a faster, more performant datastore for your IPFS node, you can use a profile like `badgerds`, but also it requires more memory. If you're running your ipfs node on a low-power device (like a raspberry-pi), you can use a profile like `low-power`. Profiles have been developed to customize your ipfs node to perform best under given conditions.
+
+### Default Profile
```

Comment:
```suggestion
## Default profile

```

Titles should follow sentence structure and only be one `#` up or down from the previous title. Leave an empty space after titles. [Read the contribution docs for more info](https://docs.ipfs.io/community/contribute/grammar-formatting-and-style/#style).